clang/test/Sema/attr-target-version.c

```diff
@@ -68,13 +68,15 @@ int __attribute__((target_version(""))) unsup1(void) { return 1; }
 void __attribute__((target_version("crc32"))) unsup2(void) {}
 
 void __attribute__((target_version("default+fp16"))) koo(void) {}
+//expected-error@-1 {{function multiversioning doesn't support feature 'default'}}
 void __attribute__((target_version("default+default+default"))) loo(void) {}
+//expected-error@-1 {{function multiversioning doesn't support feature 'default'}}
```

Comment:
Maybe this should get its own diagnostic. I could see that phrasing being really confusing.

Reply:
This diagnostic is unrelated to the patch. I believe it deserves a separate PR if at all. I am not confused by the message itself to be honest as the ACLE is clear that the attribute string is eather "default" or a list of features with '+' in between. What bothers me is the inconsistency with `target_clones`, where "default+fp16" is treated as a warning and "default" is ignored. Both attributes should either warn or error.